eg01-eg23/eg01-eg23-cycamore.xml

```diff
@@ -1,279 +1,303 @@
-        <fill_commods>
-          <val>DU2</val>
-        </fill_commods>
-        <fill_recipe>DU</fill_recipe>
-        <fill_size>1e+60</fill_size>
-        <fiss_commod_prefs>
-          <val>1</val>
-          <val>2</val>
-        </fiss_commod_prefs>
-        <fiss_commods>
-          <val>LWR Reprocessed</val>
-          <val>FR Reprocessed</val>
-        </fiss_commods>
-        <fiss_size>1e+60</fiss_size>
-        <outcommod>FR Fuel</outcommod>
-        <spectrum>fission_spectrum_ave</spectrum>
-        <throughput>1e+60</throughput>
-      </FuelFab>
-    </config>
-    <name>FR Fuel Fab</name>
   </facility>
 
-
   <facility>
+    <name>uox_reprocessing</name>
     <config>
       <Separations>
-        <feed_commod_prefs>
-          <val>2</val>
-        </feed_commod_prefs>
-        <feed_commods>
-          <val>LWR Spent Fuel</val>
-        </feed_commods>
-        <feedbuf_size>1e+100</feedbuf_size>
-        <leftover_commod>WASTE</leftover_commod>
-        <streams>
+         <feed_commods>   <val>cool_uox_waste</val> </feed_commods>
+         <feed_commod_prefs> <val>1.0</val> </feed_commod_prefs>
+         <feed_recipe>uox_used_fuel_recipe</feed_recipe>
+         <feedbuf_size>1E100</feedbuf_size>
+         <throughput>1E100</throughput>
+         <leftover_commod>reprocess_waste</leftover_commod>
+         <leftoverbuf_size>1E100</leftoverbuf_size>
+         <streams>
+          <item>
+            <commod>uox_Pu</commod>
+            <info>
+              <buf_size>1E100</buf_size>
+              <efficiencies>
+                <item>
+                  <comp>Pu</comp> <eff>.998</eff>
+                </item>
+              </efficiencies>
+            </info>
+          </item>
           <item>
-            <commod>LWR Reprocessed</commod>
+            <commod>uox_U</commod>
             <info>
-              <buf_size>1e+100</buf_size>
+              <buf_size>1E100</buf_size>
               <efficiencies>
                 <item>
-                  <comp>Pu</comp>
-                  <eff>0.99</eff>
+                  <comp>U</comp> <eff>.998</eff>
                 </item>
               </efficiencies>
             </info>
           </item>
         </streams>
-        <throughput>1e+100</throughput>
       </Separations>
     </config>
-    <name>LWR Separation</name>
   </facility>
 
-
   <facility>
+    <name>mox_reprocessing</name>
     <config>
       <Separations>
-        <feed_commod_prefs>
-          <val>2</val>
-        </feed_commod_prefs>
-        <feed_commods>
-          <val>FR Spent Fuel</val>
-        </feed_commods>
-        <feedbuf_size>1e+100</feedbuf_size>
-        <leftover_commod>WASTE</leftover_commod>
-        <streams>
+         <feed_commods>   <val>cool_sfr_spent</val> <val>cool_mox_waste</val> </feed_commods>
+         <feed_commod_prefs> <val>2.0</val> <val>1.0</val> </feed_commod_prefs>
+         <feedbuf_size>1E100</feedbuf_size>
+         <throughput>1E100</throughput>
```

Comment:
Regarding that issue:
What would you consider a realistic number for this kind of scenario?
I think there's two ways of approaching this:

Calculate the maximum enrichment (or reprocessing, source, etc.) demand by hand and use that number.
Use infinite throughput / capacity and run a successful simulation, then look at the output file to see what a 'reasonable' throughput / capacity is, then edit the input file.
I was thinking that a realistic approximation can be set after running a successful simulation
(when the recipes are set and all,). What do you think?